glados-web/templates/index.html

```diff
@@ -59,11 +61,19 @@ <h3 style="text-align: center">Client Count</h3>
             </div>
         </div>
     </div>
+    <br>
+    <div class="card shadow-sm card-box">
+        <div class="card-body">
+            <div id="my_dataviz"></div>
+        </div>
+    </div>
+    <div id="hover" style="position: absolute; opacity: 0;" ></div>
```

Comment:
```suggestion
    <div id="hover" style="pointer-events: none; position: absolute; opacity: 0;" ></div>
```

This will fix the bug I'm seeing where a node's hover-over doesn't work if it was underneath the previous hover-over

Reply:
I would also put a bit of a background on this div, maybe a bit see-through, so that the details are easier to read